README.md

```diff
@@ -98,6 +98,26 @@ subscript(index: Int) -> T {
 
 _Rationale:_ The intent and meaning of the first version is clear, and results in less code.
 
+#### Avoid redundant type declarations.
+
+When type information can be inferred from an initial value, avoid declaring it explicitly, unless doing so is an active decision to improve code legibility.
+
+```swift
+var isOnFire = false
+var lookupTable = [String: Int]()
```

Comment:
As suggested in your description, we do (overall) prefer the aesthetics of:

```swift
var lookupTable: [String: Int] = []
```

Can you change this example to use that?